.github/workflows/ci.yml

```
@@ -37,14 +37,15 @@ jobs:
 
   link-checker:
     name: Link Checker
+    timeout-minutes: 2
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
 
       - name: Link Checker
         uses: lycheeverse/lychee-action@v1
         with:
-          args: --require-https --no-progress --exclude-path apps/app/lib/social.tsx packages/ui/components/landing/Footer/config.tsx './**/*.md' './**/*.html' './**/*.json' './**/*.tsx' './**/*.ts' './**/*.js' './**/*.tex'
+          args: --require-https --no-progress --exclude-path apps/app/lib/social.tsx --exclude-path packages/ui/components/landing/Footer/config.tsx './**/*.md' './**/*.html' './**/*.json' './**/*.tsx' './**/*.ts' './**/*.js' './**/*.tex'
```

```suggestion
          args: --require-https --no-progress --exclude-path apps/app/lib/social.tsx  './**/*.md' './**/*.html' './**/*.json' './**/*.tsx' './**/*.ts' './**/*.js' './**/*.tex'
```

Should we remove Footer config from exclude file then? @CarlosHSF99 @RuiL1904

---

Although all the links in that file are valid, their purpose is to be build a larger link so I think it does make sense to exclude it.